tests/test_sipApproximation.py

```diff
@@ -425,6 +426,580 @@ def setUp(self):
+                                  M4 = -0.83497291262008522
+                                  M5 = -0.33502422292365724
+                                  M6 = -0.60881278266047245
+                                  M7 = 0
+                                  M8 = 0.79331393260752225
+                                  IM0 = -0.6623956449313968
+                                  IM1 = -0.43655355369660115
+                                  IM2 = -0.60881278266047245
+                                  IM3 = 0.55029104589365396
+                                  IM4 = -0.83497291262008522
+                                  IM5 = 0
+                                  IM6 = -0.5083421823783536
+                                  IM7 = -0.33502422292365724
+                                  IM8 = 0.79331393260752225
+                                  Form = "Full"
+                               End MatrixMap
+                            MapB =
+                               Begin CmpMap
+                                  Nin = 3
+                                  Nout = 2
+                               IsA Mapping
+                                  MapA =
+                                     Begin SphMap
+                                        Nin = 3
+                                        Nout = 2
+                                        Invert = 1
+                                     IsA Mapping
+                                        UntRd = 1
+                                        PlrLg = 0
+                                     End SphMap
+                                  MapB =
+                                     Begin CmpMap
+                                        Nin = 2
+                                     IsA Mapping
+                                        MapA =
+                                           Begin WcsMap
+                                              Nin = 2
+                                              Invert = 1
+                                           IsA Mapping
+                                              Type = "TAN"
+                                           End WcsMap
+                                        MapB =
+                                           Begin ZoomMap
+                                              Nin = 2
+                                              Invert = 0
+                                           IsA Mapping
+                                              Zoom = 57.295779513082323
+                                           End ZoomMap
+                                     End CmpMap
+                               End CmpMap
+                         End CmpMap
+                   End CmpMap
+                MapB =
+                   Begin UnitMap
+                      Nin = 2
+                      IsSimp = 1
+                   IsA Mapping
+                   End UnitMap
+             End CmpMap
+       End CmpMap
+ End FrameSet
+"""
+
```

Comment:
That's a lot of string data for file that contains code.  Mind dropping it into a separate text file?